search/__init__.py

```diff
@@ -8,4 +11,8 @@ def create_app() -> Flask:
     app = Flask(__name__)
     app.register_blueprint(get_search_blueprint())
     app.register_blueprint(get_ping_blueprint())
+
+    for code in default_exceptions:
+        app.register_error_handler(code, http_error_handler)
```

Comment:
Is it not possible to say any 4xx/5xx code?

Assume any other exception ends up as a 500 and is still handled?

Reply:
I don't think so. Here the error handler is registered for each error code. I agree that it would make sense to have something more encompassing but I don't think this is possible using nice syntax. An alternative is that I create a list of integers from 400 - 599 and register the custom handler for each integer. I'll have a closer look at the Flask code and see if I can find a work around.

Reply:
After reading the library code directly, I was able to do this. Pushing an update.